src/Components/UI/Form/Input.js

```diff
@@ -0,0 +1,22 @@
+import classes from "./Input.module.css";
+
+function Input(props) {
```

Comment:
Nothing wrong here, just wanted to share what i've seen. I've seen people use a variable they defined called ComponentProps. Benefits of this, is that you get to see what props you're expecting in each component right away.

`const InputProps = {
  input: props.input,
  label: props.label,
  inputChangeHandler: props.inputChangeHandler,
 ...props
}
`

Each solution has its pros and cons so you have to choose if you like it for yourself.